providers/cstor/cstor-disk-pool/test.yml

```diff
@@ -62,7 +78,7 @@
               args:
                 executable: /bin/bash
               register: pool_count
-              until: pool_count.stdout_lines|length == maxpool_count.stdout|int
+              until: pool_count.stdout_lines|length == diskList.stdout_lines|length
```

Comment:
@shashank855 Does this validation works for Mirrored pool?

Reply:
@dargasudarshan No for mirrored pool separate util has to be written.